react/ActionMenu/index.jsx

```diff
@@ -77,9 +94,9 @@ const ActionMenu = ({
         onClose={onClose}
         inline={shouldDisplayInline}
         anchorElRef={anchorElRef || containerRef}
-        containerElRef={containerElRef}
```

Comment:
should we log something like `containerElRef is not need anymore, you can remove it from your code`?